examples/Cargo.toml

```diff
@@ -86,27 +86,37 @@ path = "src/uds/client.rs"
 name = "uds-server"
 path = "src/uds/server.rs"
 
+[[bin]]
+name = "interceptor-client"
+path = "src/interceptor/client.rs"
+
+[[bin]]
+name = "interceptor-server"
+path = "src/interceptor/server.rs"
+
 [dependencies]
 tonic = { path = "../tonic", features = ["tls"] }
 prost = "0.6"
-
-tokio = { version = "0.2", features = ["rt-threaded", "time", "stream", "fs", "macros", "uds"] }
-futures = { version = "0.3", default-features = false, features = ["alloc"]}
+tokio = { version = "0.2", features = [
+    "rt-threaded",
+    "time",
+    "stream",
+    "fs",
+    "macros",
+    "uds"
```

Comment:
nit. Newlines on inline tables are weird. Maybe even not officially supported?

Reply:
I had a formatter plugin do this but ill revert 👍